### DISCUSSION TITLE
Table Update

### DISCUSSION THREAD

**Discussion by AcernisSupport** [ant-design-blazor/ant-design-blazor#3968] [General]
Good evening list.

I have a problem, I'm reading from a data source some lines.

The query works because if I look inside the DATA I find data 

I pass the data to the class like this

                        **headDto = data. Data.ToList(); (headDto is my class)**

but when I try to refresh the table with 

			**tHead!. ReloadData(0);**

I don't get any results

I'd like to understand where I'm going wrong.

Thank you all and good work

**Comment by ElderJames** [comment]
Hello @AcernisSupport , you need use the `OnChange` event handler to load your data and assign to DataSource.

**Comment by AcernisSupport** [comment]
Thanks Elder James for information
